testrail.py

```diff
@@ -201,15 +203,18 @@ def test_suites_update(self, testrail_project_id,
         self.session.add(suites)
         self.session.commit()
 
-    def report_test_coverage_payload(self, cases):
+    def report_test_coverage_payload(self, cases_metadata):
         """given testrail data (cases), calculate test case counts by type"""
 
         payload = []
+        cases = cases_metadata['cases']
 
         for case in cases:
+
             row = []
             suit = case['suite_id']
-            subs = case['custom_sub_test_suites']
+            subs = case.get("custom_sub_test_suites", [7])
```

Comment:
discussed with Isabel.  Gonna land this now and later add an enum.
Note, this values, once defined in TestRail become a reference and won't be changing, hence we hard-code them here.

Example:
OTHER = 7
 subs = case.get("custom_sub_test_suites", [OTHER])